lib/matplotlib/ticker.py

```diff
@@ -2884,7 +2884,8 @@ def __init__(self, n=None):
         major ticks; e.g., n=2 will place a single minor tick midway
         between major ticks.
 
-        If *n* is omitted or None, it will be set to 5 or 4.
+        If *n* is omitted or None, the value stored in rcParams will be used.
+        In case *n* is set to 'auto', it will be set to 4 or 5.
```

Comment:
```suggestion
        If *n* is set to 'auto', it will be set to 4 or 5.
```

Reply:
The "4 or 5" is not your fault, but it would be nice if we could avoid being so vague.  What does 4 or 5 depend on?